acceptance/tests/base/dsl/helpers/web_helpers/web_helpers_test.rb

```diff
@@ -0,0 +1,66 @@
+require 'helpers/test_helper'
+require 'webrick'
+require 'webrick/https'
+
+test_name 'dsl::helpers::web_helpers #link_exists?' do
+
+  http_cmd = "ruby -rwebrick -e'WEBrick::HTTPServer.new(:Port => 80, :DocumentRoot => Dir.pwd).start' > /tmp/mylogfile 2>&1 &"
+  host = hosts[0]
+  on(host, http_cmd)
+  on(host, 'mkdir tmp')
+  create_remote_file(host, 'tmp/test.txt', 'test')
```

Comment:
For all this temp file/dir stuff, there's convenience methods on `host` for doing it, something like `host.tmpdir('my-test')` that results in `on(host, "mkdir <proper temp directory for OS>/my-test")`.  

There might be a couple of methods for working with temp files/dirs in beaker, probably worth a look.